### DISCUSSION TITLE
Adding a new document in the Unreleased version only is causing an error

### DISCUSSION THREAD

**Discussion by Sotty75** [facebook/docusaurus#10146] [Q&A]
Hi,
i am working on Docusurus with enabled versioning, and tried to add a new section to the site in the currently unreleased version of the documentation. To do this i added the following item in the navBar of docusaurus config file:

```
 {
                        docId: 'Architectural Docs/_index',
                        label: 'Architecture',
                        position: 'left',
                        type: 'doc',
                    },
```

the problem is that the site does not work as it expects this to be present also in the latest version of the documentation, which should not contain this document until a new release is published. 
Is there a way to workaround this problem?

![image](https://github.com/facebook/docusaurus/assets/1228599/f0fd3e2a-adb3-4fc8-94b5-695b7652ebb6)

**Comment by slorber** [comment]
This doc does not exist in version 4.0 so you can't use a "doc" navbar item which is mostly useful for linking to versioned docs existing in all versions. 

You can replace that navbar item with a simple link such as `{to: '/docs/myDoc', label: 'MyDoc'}` instead

I understand it's a bit confusing, maybe we could allow to pass a "version" attribute in the doc navbar item so that you can still use it in case the doc only exists in a single version.
